### PR TITLE
RANGER-3968:The plugin cannot download the latest policy

### PR DESCRIPTION
**Pull request by Hoo199212**
Add transaction timeout settings

### CODE REVIEW

**Comment by Hoo199212** [comment]
@bhavikpatel9977 please review, thank you so much!

**Comment by mneethiraj** [comment]
@Hoo199212  - what happens in environments having large number of policies, where loading can take more than 30 seconds? Wouldn't this patch result in loading to fail every time?

**Comment by Hoo199212** [comment]
> 
In our production environment, the cache update thread has been stuck, the plugin can not get the latest permission information for a long time (perhaps more than 1 day)，But once the server is restarted, it will be normal immediately. The timeout allows the thread to release the lock. The transaction time can be set, if the data volume is large and takes a long time to load, we can set a reasonable value

File: security-admin/src/main/resources/conf.dist/ranger-admin-default-site.xml [hunk 1]
```diff
@@ -681,4 +681,8 @@
 		<name>ranger.allow.kerberos.auth.login.browser</name>
 		<value>false</value>
 	</property>
+	<property>
+		<name>ranger.policy.retriever.transaction.timeout</name>
```
Comment (pan3793) [inline security-admin/src/main/resources/conf.dist/ranger-admin-default-site.xml:685]: What's the time unit? Could you please follow the existing configuration key naming style?
`<name>ranger.resource.lookup.timeout.value.in.ms</name>`

**Pull request closed without merging**